src/features/classification/security-controls/mixins.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../../styles/variables';
+
+$scrollbar-width: 7px;
+
+@mixin BackgroundlessScrollbar {
+    // Currently only supported by Firefox.
+    // For Chrome/Safari we use the styles below
+    scrollbar-color: $bdl-gray-50 transparent;
```

Comment:
How does this look on IE 11?

Reply:
Thanks for bringing this up! Confirming with design team, I reverted the update here.